invenio_communities/views/communities.py

```diff
@@ -432,11 +434,28 @@ def communities_home(pid_value, community, community_ui):
     if not permissions["can_read"]:
         raise PermissionDeniedError()
 
+    recent_uploads = current_community_records_service.search(
```

Comment:
Perhaps we need to implement this in `invenio-rdm-records` instead. Community records "service" is a "proxy" for rdm-records to fetch data from the `communities` service.